libs/sdk-core/Cargo.toml

```diff
@@ -48,9 +48,8 @@ tonic = { version = "^0.8", features = [
     "tls-webpki-roots",
 ] }
 lazy_static = "^1.4.0"
-lightning = "0.0.118"
-# Keep in sync with version used in gl-client
-lightning-invoice = "0.26.0"
+# Keep in sync with the version used by gl-client
+lightning = "0.0.116"
```

Comment:
This is changed from 0.0.118 to 0.0.116. Why is that?
Can we also do the `lightning` crate in the same construct?

Reply:
> This is changed from 0.0.118 to 0.0.116. Why is that?

This is because GL recently consolidated two `lightning-invoice` versions, who each were compatible with different `lightning` versions. The winning pair was `lightning-invoice` 0.24.0 / `lightning` 0.0.116, as used in VLS ([link](https://gitlab.com/lightning-signer/validating-lightning-signer/-/blob/v0.11.0-rc.1/vls-core/Cargo.toml?ref_type=tags#L128) / [link](https://gitlab.com/lightning-signer/validating-lightning-signer/-/blob/v0.11.0-rc.1/vls-core/Cargo.toml?ref_type=tags#L110)).

We had both 0.0.116 and 0.0.118 before, but now only one of them is compatible with the GL code, which is 0.0.116.

So we're now using the same version as GL and VLS.

> Can we also do the lightning crate in the same construct?

Would first need to ask GL to re-export the VLS `lightning` dependency. Added a PR for that: https://github.com/Blockstream/greenlight/pull/359

Once that's reviewed and merged, I will add a PR to the SDK.